cartography/intel/aws/ecr.py

```diff
@@ -75,21 +75,17 @@ def transform_ecr_repository_images(repo_data: Dict) -> List[Dict]:
     """
     repo_images_list = []
     for repo_uri, repo_images in repo_data.items():
-        filtered_imgs = []
         for img in repo_images:
             if 'imageDigest' in img and img['imageDigest']:
-                filtered_imgs.append(img)
+                img['repo_uri'] = repo_uri
```

Comment:
This line allows us to flatten the data structure loaded to neo4j to just a list of dicts.